console-subscriber/src/lib.rs

```diff
@@ -248,7 +248,7 @@ impl TasksLayer {
 }
 
 impl TasksLayer {
-    pub const DEFAULT_EVENT_BUFFER_CAPACITY: usize = 1024 * 10;
```

Comment:
was this change supposed to be part of this PR? In principle, the fix you put in should still work even without scaling up the default event-buffer capacity, right?

Reply:
i can back out those changes, yeah; i was just messing with things trying to figure out what was going on